lib/t/accounts/age_preference.ex

```diff
@@ -0,0 +1,11 @@
+defmodule T.Accounts.AgePreference do
+  use Ecto.Schema
+
+  @primary_key false
+  @foreign_key_type Ecto.Bigflake.UUID
+  schema "age_preferences" do
+    field(:user_id, Ecto.UUID, primary_key: true)
```

Comment:
Something's wrong with your formatter setup, `field` macros don't require brackets. I'ts not a problem here, but

Reply:
yeah, very weird
I even tried setting default formatter, but it didn't help:
![image](https://user-images.githubusercontent.com/16062322/140926029-3b20286e-5694-4994-ab6b-14ed3cf3c1e1.png)

Reply:
my local formatter.exs seem to be the same as here in master:
![image](https://user-images.githubusercontent.com/16062322/140927781-62241bfa-298d-42c3-a236-4a575cc0188a.png)

maybe some deps bump led to this? 🤔